### PR TITLE
Bugfix/make rows clickable

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: src/components/SortableTable/index.tsx [hunk 1]
```diff
@@ -119,13 +119,27 @@ export const SortableTable = <DataItem,>({
...
+                <>
+                  {hideNumberColumn ? null : <td>{index + 1}</td>}
+                  {renderItemRow(item)}
+                </>
+              );
+
+              return (
+                // eslint-disable-next-line react/no-array-index-key
+                <tr key={`row-${index}`}>
+                  {item.href ? (
```
Comment (piyalbasu) [inline src/components/SortableTable/index.tsx:133]: thoughts on this API?

The idea is if DataItem contains a magic prop called `href`, it'll wrap the whole row in an anchor.

My thought is a) this is something we'll likely need frequently in tables and b) best to use an anchor to so users/screen readers can see where it links to

File: src/components/TopList/index.tsx [hunk 2]
```diff
@@ -70,10 +70,7 @@ export const TopList = ({ aggregatedPoolData }: TopListProps) => {
           </div>
           <div className="TopList__pool__row">
             {topPools.map((pool) => (
-              <div
-                className="TopList__pool__row__item"
-                key={getPoolName(pool.assetCodes)}
-              >
+              <div className="TopList__pool__row__item" key={pool.id}>
```
Comment (piyalbasu) [inline src/components/TopList/index.tsx:73]: unrelated to other things in this PR - this is was throwing a warning bc of multiple pools with the same name

File: src/components/SortableTable/index.tsx (hunk 1 above)
Comment (quietbits) [reply src/components/SortableTable/index.tsx:133]: I like this approach. The only change I would make is maybe adding `onClick` event on the anchor. Right now it's not using the router but does the full page refresh when going to the link and back. What do you think?

File: src/components/TopList/index.tsx (hunk 2 above)
Comment (quietbits) [reply src/components/TopList/index.tsx:73]: Interesting, I thought there could only be a single pool pair.

**Review by quietbits — APPROVED** [review]

File: src/components/SortableTable/index.tsx (hunk 1 above)
Comment (piyalbasu) [reply src/components/SortableTable/index.tsx:133]: Ah good cool - i should actual be using the router's `Link` component here

**Review by quietbits — APPROVED** [review]

**Pull request merged**